employee-onboarding-A2A-reference-implementation/agents/approval_agent/agent.py

```
@@ -0,0 +1,215 @@
+
+        # Get actor token: 3-step flow using TEApp credentials (OAuth app) + agent credentials (authn)
+        # Then RFC 8693 exchange: OBO token as subject, agent actor token as actor
+        try:
+            from src.auth.asgardeo import get_asgardeo_client
+            asgardeo = get_asgardeo_client()
+            actor = await asgardeo._fetch_agent_actor_token(
+                client_id=self.settings.token_exchanger_client_id,
+                client_secret=self.settings.token_exchanger_client_secret,
+                agent_id=self.agent_id,
+            )
+            log_and_broadcast(f"\n[APPROVAL_AGENT_ACTOR_TOKEN]:")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)